Tests/UI/LambdaTestPlayground/TestSpecialPage.spec.js

```diff
@@ -37,33 +38,4 @@ test('Verify the primary headers are displayed on the Special page', runTest(asy
   expect(productCompareLink).toBe(true);
   const copyrightText = await specialMenu.isCopyrightTextVisible();
   expect(copyrightText).toBe(true);
-}));
-
-  // Cameras Menu
-test('Verify a price range can be selected from the fields', runTest(async({ page }) => {
```

Comment:
did you meant to delete these tests?

Reply:
Yes I did since they are now in the TestCamerasPage.spec.js file instead.